src/app/page/Signup/Signup.js

```diff
@@ -7,31 +7,70 @@ import Button from "../../component/Button";
 import { FormWrapper, HeaderText, DescriptionText, PaddedDiv } from "./Signup.style";
 
 const SignupPage = () => {
+  const [form, setForm] = useState({
+    fullName: "",
+    email: "",
+    phoneNumber: "",
+    zipCode: "",
+    errors: "",
+  });
+
   const handleLoginCTAClick = (e) => {
     e.preventDefault();
 
     window.location = "/login";
```

Comment:
not part of your changes though but can with use the `history` props given by react-router as window.location will reload the whole page

Reply:
That's a much better approach. I'll get on it. Thanks

Reply:
I got it working with props. 

    props.history.push('/login');

I used withRouter but I think I can pass in the history prop from the parent. What do you suggest?

Reply:
I think you don't need withRouter, history is already passed from the Router component
Remove withRouter and console.log(prop.history). You will see it available

Reply:
I tried console.log(props.history) and got undefined. I looked at App.tsx and noticed this 

  <Route path="/signup">
      <SignupPage />
  </Route>

If I change that to 

  <Route path="/signup" component={Signup}  />

It should pass in the history props right?

Reply:
Yeah